source/configuration_manual/namespace/index.rst

```diff
@@ -555,109 +563,156 @@ on and setting ``subscriptions = no`` for the other namespaces:
 
 .. _mailbox_settings:
 
-===================
-Mailbox settings
-===================
+================
+Mailbox Settings
+================
 
-One can assign SPECIAL-USE `RFC 6154 <http://www.faqs.org/rfcs/rfc6154.html>`_
-tags and specify, which mailboxes to create and/or subscribe to automatically.
+Mailbox configuration is defined within a dovecot configuration block, inside
+of a ``namespace`` block, with the format::
 
-Support for IMAP \Important SPECIAL-USE flag `RFC 8457 <http://www.faqs.org/rfcs/rfc8457.html>`_
+  mailbox <mailbox-name> {
+    [... mailbox settings ...]
+  }
 
-  .. versionadded:: v2.3.10
+The mailbox-name specifies the full mailbox name; if it has spaces, you can
+put it into quotes.
 
-The autocreated mailboxes are created lazily to disk only when accessed for the
-first time. The autosubscribed mailboxes aren't written to subscriptions file,
-unless SUBSCRIBE command is explicitly used for them.
+Settings
+========
+
+.. _setting-namespace-mailbox_auto:
+
+``namespace/mailbox/auto``
+--------------------------
 
-The mailbox section name specifies the mailbox name. If it has spaces, you can
-put it in `quotes`. The mailbox settings are:
+- Default: ``no``
+- Values:  ``create``, ``no``, or ``subscribe``
 
-* auto: Autocreate/subscribe mailbox?
+Autocreate and/or subscribe to the mailbox?  Options:
 
- * no: Neither
- * create: Autocreate, but don't autosubscribe
- * subscribe: Autocreate and autosubscribe
+============== ==================================
+Value          Description
+============== ==================================
+``create``     Autocreate but don't autosubscribe
+``no``         Don't autocreate or autosubscribe
+``subscribe``  Autocreate and autosubscribe
```

Comment:
"no" looks weird in the middle. Would make sense to me more being the first. I guess you did it alphabetically .. but logical sorting seems cleaner to me.

Reply:
See discussion above.  It's already clearly indicated that "no" is the default.  To me, the purposes of these "settings" entries is not to discuss which setting SHOULD be used - that should be discussed elsewhere.  Setting entries to me should plainly list what the setting does and all available options.